src/_includes/navbar.njk

```diff
@@ -11,8 +11,8 @@
   </div>
   <div id="menu"
     class="w-full sm:w-auto self-end sm:self-center hidden sm:flex flex-col sm:flex-row items-center h-full py-1 pb-4 sm:py-0 sm:pb-0">
-    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center no-underline py-2 mx-2" href="{{ env.baseUrl }}coc">Code Of Conduct</a>
-    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center no-underline py-2 mx-2" href="https://in.pycon.org/blog/">Blog</a>
+    
+    <a class="text-gray-800 font-bold hover:text-purple-600 text-lg text-center w-full no-underline py-2 whitespace-nowrap ml-6" href="{{ env.baseUrl }}coc">Code Of Conduct</a>
```

Comment:
why `whitespace-nowrap` and extra `ml-6`?

Please keep the current navbar changes only?